self_hosted/main.jou

```diff
@@ -467,6 +483,13 @@ def main(argc: int, argv: byte**) -> int:
         compiler.process_imports_and_exports()
         compiler.typecheck_stage3_all_files()
 
+        mainfile = find_file(compiler.files, compiler.nfiles, args.main_path)
+        assert mainfile
+
+        if (not check_main_function(&mainfile.ast)):
```

Comment:
Nit: Jou does not need as much parentheses as C.
```suggestion
        if not check_main_function(&mainfile.ast):
```

Reply:
Sorry, get used to it